### PR TITLE
Migrate export rollout

### PR DESCRIPTION
**Pull request by benrudolph**
@NoahCarnahan @czue it's show time!

cc: @dannyroberts

### CODE REVIEW

File: corehq/apps/export/management/commands/migrate_exports.py [hunk 1]
```diff
@@ -3,26 +3,78 @@
...
+
+                has_skipped_tables = any(map(lambda meta: bool(meta.skipped_tables), metas))
+                has_skipped_columns = any(map(lambda meta: bool(meta.skipped_columns), metas))
+                is_remote_app_migration = any(map(lambda meta: bool(meta.is_remote_app_migration), metas))
+                if has_skipped_tables or has_skipped_columns:
+                    print 'Skipping {} because we would have skipped columns'.format(domain)
+                    skipped_domains.append(domain)
+                    continue
+
+                if is_remote_app_migration:
```
Comment (benrudolph) [inline corehq/apps/export/management/commands/migrate_exports.py]: want to make the uneditable user defined columns before we migrate these

**Comment by czue** [comment]
:tada: :clap: :pray:

**Comment by czue** [comment]
looks good on :white_check_mark: 

is there an edge case where a domain edits an export between when we start the migration and when we actually flip them? guessing we don't care, just occurred to me. if we were being paranoid we could probably figure something out (e.g. lock editing, double check the revs before flipping the switch, etc.)

**Comment by NoahCarnahan** [comment]
wooooo 🎉

**Comment by benrudolph** [comment]
> is there an edge case where a domain edits an export between when we start the migration and when we actually flip them?

yes you're right, there's a window where they could edit the export and save it during the time we are migrating. however this would cause an exception to be raised when we try to resave the export schema. adding logic now to ensure we don't continue to migrate the domain if we fail on migrating an export

**Comment by benrudolph** [comment]
https://github.com/dimagi/commcare-hq/pull/13063/commits/332a8c10eea8bb9780f445a0f3d9bd57adbba4e4 this should handle the case when the migration fails (and the scenario you mentioned) and just skip that domain

**Comment by czue** [comment]
sweet thanks.

**Comment by benrudolph** [comment]
🙏 🙏 🙏

**Comment by benrudolph** [comment]
✔️

**Pull request merged**